### PR TITLE
[kubernetes OTEL] Add kubernetes OTEL package

### PR DESCRIPTION
**Pull request by tetianakravchenko**
## Proposed commit message

Please explain:

- WHAT: Add a new package - kubernetes OTEL of type content to ship assets for the EDOT installation
- WHY:  provide assets for the EDOT installation

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- 

## Screenshots
Screenshot 2024-09-20 at 12 34 06

Screenshot 2024-09-20 at 12 34 19

Screenshot 2024-09-20 at 12 34 32

Tags:
Screenshot 2024-09-20 at 12 35 12

### CODE REVIEW

File: packages/kubernetes_otel/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+format_version: 3.4.0
+name: kubernetes_otel
+title: "Kubernetes OTEL"
+version: 0.0.1
+description: "Assets to monitor Kubernetes clusters with Elastic Agent."
```
Comment (gizas) [inline packages/kubernetes_otel/manifest.yml:5]:
```suggestion
description: "Assets to monitor Kubernetes clusters with Elastic Distribution of OpenTelemetry Collector"
```

File: packages/kubernetes_otel/kibana/tags.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+- text: OTEL
```
Comment (gizas) [inline packages/kubernetes_otel/kibana/tags.yml:1]: Maybe to have a Kubernetes tag as well?

Comment (tetianakravchenko) [reply packages/kubernetes_otel/kibana/tags.yml:1]: 2 default tags are `Managed` and the name of the integration:
Screenshot 2024-09-16 at 15 41 32
do you mean to add one more `Kubernetes`?

Comment (tetianakravchenko) [reply packages/kubernetes_otel/kibana/tags.yml:1]: TODO: rename OTEL -> OpenTelemetry

**Review by gizas — APPROVED** [review]

File: packages/kubernetes_otel/kibana/tags.yml (hunk 2 above)
Comment (tetianakravchenko) [reply packages/kubernetes_otel/kibana/tags.yml]: done https://github.com/elastic/integrations/pull/11137/commits/531e6f87adf82e95f408b68e8b1060f2084a99e3

File: packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-70c709ff-59b9-43b7-8ef4-4c34d8890bde.json [hunk 3]
```diff
@@ -0,0 +1,4668 @@
...
+                                                },
+                                                "95e56b61-fe65-4ac4-81aa-d9608a43162d": {
+                                                    "customLabel": true,
+                                                    "dataType": "number",
+                                                    "filter": {
+                                                        "language": "kuery",
+                                                        "query": "\"metrics.k8s.node.cpu.usage\": *"
+                                                    },
+                                                    "isBucketed": false,
+                                                    "label": "CPU usage Pct",
```
Comment (ChrsMark) [inline packages/kubernetes_otel/kibana/dashboard/kubernetes_otel-70c709ff-59b9-43b7-8ef4-4c34d8890bde.json:1170]: Is this dashboard finalized or just a work-in-progress/placeholder? 
Just spotted that the specific `Usage PCT` label seems incorrect as noted at https://github.com/elastic/integrations/pull/10910#discussion_r1756257787. Didn't check for other inconsistencies in that PR.

**Comment by AlexanderWert** [comment]
@tetianakravchenko Does this PR replace the one in https://github.com/elastic/integrations/pull/10910 ?

If so, can we please make sure [the comments](https://github.com/elastic/integrations/pull/10910#pullrequestreview-2299392318) @ChrsMark left there are considered here? Maybe in a followup PR?

**Pull request merged**